src/SAWScript/Crucible/LLVM/Builtins.hs

```diff
@@ -1564,7 +1561,9 @@ crucible_fresh_var bic _opts name lty =
      let dl = Crucible.llvmDataLayout (ccTypeCtx cctx)
      case cryptolTypeOfActual dl lty' of
        Nothing -> throwCrucibleSetup loc $ "Unsupported type in crucible_fresh_var: " ++ show (L.ppType lty)
-       Just cty -> Setup.freshVariable sc name cty
+       Just cty ->
+         do tec <- Setup.freshVariable sc name cty
+            liftIO $ typedTermOfExtCns sc tec
```

Comment:
extract in a helper

Reply:
Done in 694a3d5d90f27e956532d4668d5a7b73da68e5ba